client/src/app/components/tool-container/tool-header/tool-header.component.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../../../assets/sass/variables";
+
+.toolHeaderBtn {
+    width: 8%;
```

Comment:
Avoid setting width and height on buttons, because it can lead to situations where the text no longer fits inside the button. If you set a fixed width/height in pixels, text will overflow if the user increases the font-size. If you set a percent width/height, at a certain point the text will also overflow because the button is too small. It's better to either use [bootstrap sizes](http://v4-alpha.getbootstrap.com/components/buttons/#sizes) like `btn-sm`, `btn-lg` etc, or to set a custom padding on the button to make it smaller or larger.